microservices/ga/datamgr/README.md

```diff
@@ -13,110 +16,133 @@ The build system requires git source code management.  If the directory you choo
 
-    films -v
+    $ films -v
     {"Version": "1.0.0", "Build": "8755e7f", "GitTag": "v0.0alpha"}
 
 VERSION := 1.0.0
 ----------------
-The version variable is set based on the value you enter in your definitions file
+The version variable is set based on the value you enter in your definitions file.
 
 BUILD := $(shell git rev-parse --short HEAD)
 --------------------------------------------
-The build is set to the commit id of your current HEAD
+The build is set to the commit ID of your current git HEAD.
 
 GIT_TAG := $(shell git describe)
 --------------------------------
 GIT_TAG set using the most recent tag if any.  You can add a tag with:
+
     git tag -a "mytag" -m "message about the tag."
 Git push doesn't include tags.  To push tags to the origin use:
+
     git push origin --tag
 
 ## roadctl
-The roadctl is modeled after kubectl. Use roadctl help for a list of top
-level commands.  This may be different for a specific command or it's 
-sub-commands.
+The roadctl command is modeled after kubectl.
+Use "roadctl help" for a list of top level commands.
+This may be different for a specific command or it's sub-commands.
 
 ### Top level help
-roadctl help
-roadctl allows you to work with the PavedRoad CNCF low-code environment and the associated CI/CD pipeline
+    roadctl help
+
+General help output:
+
+```
+roadctl allows you to work with the PavedRoad CNCF low-code environment and the associated CI
 
   Usage: roadctl [command] [TYPE] [NAME] [flags]
 
   TYPE specifies a resource type
   NAME is the name of a resource
   flags specify options
 
-    Usage:
-      roadctl [command]
-    
-    Available Commands:
-      apply       Apply configuration to named resource
-      config      manage roadctl global configuration options
-      create      create a new resource
-      delete      delete a resource
-      deploy      deploy a service
-      describe    describe provides detailed information about a resource
-      doc         Generate documentation for your service
-      edit        edit the configuration for the specified resource
-      events      View events
-      explain     return documentation about a resource
-      get         get an existing object
-      help        Help about any command
-      logs        return logs for a resource
-      replace     Delete and recreate the named resource
-      version     Print the current version
-    
-    Flags:
-          --config string   Config file (default is $HOME/.roadctl.yaml)
-          --debug string    Debug level: info(default)|warm|error|critical (default "d")
-      --format string   Output format: text(default)|json|yaml (default "f")
-  -h, --help            help for roadctl
+Usage:
+  roadctl [command]
+
+Available Commands:
+  apply       Apply configuration to named resource
+  completion  Generate completion scripts on stdout
+  config      manage roadctl global configuration options
```

Comment:
I thought we were going to change all the Usage commands to start with uppercase, aka Manage roadctl global config...